setup/www/tools/promote/promote_nightly.sh

```diff
@@ -7,12 +7,7 @@ site=$1
 __dirname="$(CDPATH= cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
 if [ "X$site" != "Xiojs" ] && [ "X$site" != "Xnodejs" ]; then
-  echo "Usage: promote_nightly.sh < iojs | nodejs > <version>"
-  exit 1
-fi
-
-if [ "X$2" == "X" ]; then
-  echo "Usage: promote_nightly.sh < iojs | nodejs > <version>"
+  echo "Usage: promote_nightly.sh < iojs | nodejs >"
```

Comment:
this fix is just something that has been sitting on the server but not included here